scripts/README.md

```diff
@@ -0,0 +1,34 @@
+# Deployment
+
```

Comment:
Not sure using a centralized method for deploying is going to be in our most-ideal interests.

Reply:
The service doesn't own the contract, so it's still super secure! And we are inherently trusting a centralized source (Firebase hosting) to serve up our HTML.

Reply:
Agreed!